sbt-compatibility-rules/src/main/scala/sbtcompatibilityrules/SbtCompatibilityRulesPlugin.scala

```diff
@@ -7,11 +7,11 @@ object SbtCompatibilityRulesPlugin extends AutoPlugin {
   override def trigger = allRequirements
 
   object autoImport {
-    val compatibilityRules = taskKey[Seq[ModuleID]]("")
+    val dependencyCompatibilityRules = taskKey[Seq[ModuleID]]("""Compatibility rules for the library dependencies (e.g. "org.scala-lang" % "scala-compiler" % "strict")""")
   }
   import autoImport._
 
-  override def buildSettings = Def.settings(
-    compatibilityRules := Seq.empty
+  override def globalSettings = Def.settings(
```

Comment:
Default values should be scoped globally - https://www.scala-sbt.org/1.x/docs/Plugins-Best-Practices.html#Provide+default+values+in